src/main/pages/setup-docker/docker-multi-user.adoc

```diff
@@ -8,56 +8,59 @@ folder: setup-docker
 ---
 
 [id="system-requirements"]
-== System Requirements
+== System requirements
 
-* Minimum 4GB RAM (just for the 3 Che containers and one 2GB workspace)
-* 2 CPU
-* Ports 8080, 5050 and the (https://en.wikipedia.org/wiki/Ephemeral_port)[ephemeral port] range publicly available for inbound connections (`32768-65535`)
+* Minimum 4GB RAM (for the three Che containers and one 2GB workspace)
+* 2 CPUs
+* Ports 8080, 5050, and the https://en.wikipedia.org/wiki/Ephemeral_port[ephemeral port] range publicly available for inbound connections (`32768-65535`)
 
 [id="known-issues"]
-== Known Issues
+== Known issues
 
-Currently, when running Che Multi User on Windows, PostgreSQL has issues with file permissions that prevents DB from starting. As a workaround, mount a local storage path that is not a Win file system mount:
+Currently, when running Che multi-user on Windows, PostgreSQL has issues with file permissions that prevents the database from starting. As a workaround, mount a local storage path that is not a WIN file system mount:
 
 `-v /chedata/:/data`
 
-We generally do not recommend running Che multi user on Windows for production use.
+It is not recommended to run Che multi-user on Windows for production use.
 
 [id="run-syntax"]
-== Run Syntax
+== Running Che multi-user 
 
+Following is the command to run Che multi-user:
 ----
 docker run -it -e CHE_MULTIUSER=true -e CHE_HOST=${EXTERNAL_IP} -v /var/run/docker.sock:/var/run/docker.sock -v ~/.che-multiuser:/data eclipse/che start
 ----
 
-`~/.che-multiuser` is any local path. This is where Che data and configuration will be stored.
+Here:
 
-`${EXTERNAL_IP}` should be a public IP accessible to all users who will access the Che instance. You may drop `CHE_HOST` env if you are running Che locally and will access it from within the same network. In this case, Che CLI will attempt to auto-detect your server IP. However, auto-detection may produce erroneous results, especially in case of a complex network setup. If you run Che as a cloud server, i.e. accessible for external users we recommend explicitly providing an external IP for `CHE_HOST`.
+* `~/.che-multiuser` is any local path. The Che data and configuration is stored at this location.
 
-You *MUST* access multi-user Che using hostname/IP specified in `CHE_HOST` or manually add redirect URIs and Web Origins to link:user-management.html#che-and-keycloak[Keycloak client].
+* `${EXTERNAL_IP}` should be a public IP accessible to all users who will access the Che instance. If you are running Che locally and will access it from within the same network, you may omit the `CHE_HOST` environment variable from the command. In this case, Che CLI will attempt to auto-detect your server IP. However, auto-detection may produce erroneous results, especially in case of a complex network setup. If you run Che as a cloud server that is accessible by external users, we recommend explicitly providing an external IP for the `CHE_HOST` variable.
```

Comment:
_However, auto-detection may produce erroneous results, especially in case of a complex network setup. If you run Che as a cloud server that is accessible by external users, we recommend explicitly providing an external IP for the `CHE_HOST` variable._

* ditch 'however'
* turn into an admonition?

Reply:
Added a note. Thanks for the suggestion!